### PR TITLE
Removed extra unused template files

### PR DESCRIPTION
**Pull request by Sid911**
# Changes done in this Pull Request
- Deleted the extra Template files
- Kept the elements section for referring purpose about using different elements

## Description / Changes
- Removed the extra unused files

### CODE REVIEW

**Comment by Sid911** [comment]
Conflicts removed

**Comment by ammarbinfaisal** [comment]
squash commits

**Comment by Sid911** [comment]
@ammarbinfaisal I will get that done

File: index.html [hunk 1]
```diff
@@ -541,7 +533,6 @@ <h1>Students</h1>
             </a>
             {% endif %} {% if student.facebook %}
 
-
             <a href="https://facebook.com/{{ student.facebook }}" class="icon-a" target="_blank">
```
[the remark this answers is not in the document]
Comment (pythongiant) [reply index.html]: Please checkout the file and remove it from commit

File: css/theme.css [hunk 2]
```diff
@@ -65,6 +65,9 @@ body.boxed-layout,
   left: 0;
   right: 0;
 }
+.extra_padding_b{
```
[the remark this answers is not in the document]
Comment (pythongiant) [reply css/theme.css]: Please remove it from the commit

**Comment by Sid911** [comment]
@fragm3 can this be merged? Or there are any problems?

**Comment by ammarbinfaisal** [comment]
there are still two commits squash them into one :smile:

**Comment by fragm3** [comment]
@Sid911 yes, once more mentors review it.

**Pull request merged**